purchase_lead_hour/models/stock_orderpoint.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from dateutil import relativedelta
+
+from odoo import fields, models
+
+
+class StockOrderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    def _get_product_context(self):
+        # Since product.supplierinfo delay is now a float, we need to get the
+        # available qty as of now + delay
+        res = super()._get_product_context()
+        if not self.product_id or not self.location_id:
+            return res
+        seller = fields.first(self.product_id._select_seller(quantity=None))
```

Comment:
```suggestion
        seller = self.product_id._select_seller(quantity=None)
```

`_select_seller` never returns more than 1 record